integration_test/third_party_apps_data/applications/mssql/metadata.yaml

```diff
@@ -38,12 +38,11 @@ expected_metrics:
   monitored_resource: gce_instance
   labels: {}
 # v2 metrics
-- type: workload.googleapis.com/sqlserver.user.connection.count
+- type: workload.googleapis.com/sqlserver.user.connection.count # if no user connection may not be present
```

Comment:
[Optional] You could have left it after `optional: true` as well.

Reply:
@igorpeshansky just removed this I think everything should be good to go now 👍